### PR TITLE
overlays: add ppu compile notification and touch existing compile notifications

### PR DESCRIPTION
**Pull request by Megamouse**
- Adds PPU compile notifications. We originally used the progress dialog for this, but due to the non-working coexistence with the cellMsgDialog this is generally disabled during gameplay.
This missing dialog added to the confusion surrounding the Castlevania issue, which is seemingly stuck but actually compiles a ton of PPU modules.
- Additionally, we now touch/refresh existing shader and ppu compile notifications.
They used to go through a whole fade in and fade out cycle before a new one was shown.
This means that even if stuff was constantly compiled, the notification kept popping in and out instead of persisting.

image

### CODE REVIEW

File: rpcs3/Emu/RSX/Overlays/overlay_message.cpp [hunk 1]
```diff
@@ -122,8 +127,13 @@ namespace rsx
 			}
 			else if (time + u64(m_fade_out_animation.duration * 1'000'000) > m_expiration_time)
 			{
+				m_fade_out_animation.active = true;
 				m_fade_out_animation.update(rsx::get_current_renderer()->vblank_count);
 			}
+			else if (m_fade_out_animation.active)
+			{
+				m_fade_out_animation.reset(rsx::get_current_renderer()->vblank_count);
```
Comment (kd-11) [inline rpcs3/Emu/RSX/Overlays/overlay_message.cpp:135]: Isn't the snapping effect jarring? Makes more sense to reverse the animation during reset though that is harder to do. Maybe add a TODO or something so we don't forget.

Comment (Megamouse) [reply rpcs3/Emu/RSX/Overlays/overlay_message.cpp:135]: It's more jarring to have an overlay constanty fading in and out xD

Comment (kd-11) [reply rpcs3/Emu/RSX/Overlays/overlay_message.cpp:135]: It's really not. A soft fade vs pop-in is a no contest.
Also think about translation animations. A card is sliding out to the edge of the screen then "snaps" back into place.

Comment (Megamouse) [reply rpcs3/Emu/RSX/Overlays/overlay_message.cpp:135]: As far as I can tell they just fade out. Feel free to make some sort of card shuffle animation.

Comment (kd-11) [reply rpcs3/Emu/RSX/Overlays/overlay_message.cpp:135]: It should still reverse if we're beyond some point even with fade-out. The card may be 99% gone and almost entirely see-through, makes sense to just fade back in.
Obviously you refuse do it, I'll add it in after this is merged.

Comment (Megamouse) [reply rpcs3/Emu/RSX/Overlays/overlay_message.cpp:135]: I don't refuse to do it, I just haven't had an issue with the way it looks.

Comment (kd-11) [reply rpcs3/Emu/RSX/Overlays/overlay_message.cpp:135]: You just have to restart the fade-in animation as well and borrow the remaining time from the fadeout to be its progress so far.

Comment (Megamouse) [reply rpcs3/Emu/RSX/Overlays/overlay_message.cpp]: done

**Pull request merged**